uber/models.py

```diff
@@ -757,7 +757,7 @@ def model_mixin(cls, model):
         for name in dir(model):
             if not name.startswith('_'):
                 attr = getattr(model, name)
-                if name in target.__table__.c:
+                if hasattr('target', '__table__') and name in target.__table__.c:
```

Comment:
About 2 weeks ago I updated this function to allow for proper overriding of columns (rather than just adding new columns to existing model objects, which already worked).  I accidentally broke this by adding an implicit assumption that everything we're mixing in would have a ``__table__`` attribute.  Whoops!

By checking for that attribute, we'll once again do the correct thing for ``SessionMixin`` additions in plugins.